content/kots-cli/pull.md

```diff
@@ -24,6 +24,7 @@ This command supports all [global flags](/kots-cli/global-flags/) and also:
 | `--exclude-admin-console` |  bool  |  set to true to exclude the admin console _(only valid when `[upstream-uri]` points to a replicated app)_ |
 | `--exclude-kots-kinds` |  bool  | set to true to exclude rendering kots custom objects to the base directory _(default `true`)_ |
 | `-h, --help`  |          |  help for pull |
+| `--helm-version` | string | the Helm version with which to render the Helm Chart _(default `"v2"`)_ |
```

Comment:
I'm thinking we should probably indicate that it's beta here, and in the CLI.

Reply:
this is just the cli output